Source/FakeItEasy/Core/FakeManager.AutoFakePropertyRule.cs

```diff
@@ -11,7 +11,7 @@ public partial class FakeManager
         private class AutoFakePropertyRule
             : IFakeObjectCallRule
         {
-            public FakeManager FakeManager { get; set; }
+            public FakeManager FakeManager { private get; set; }

```

Comment:
The write-only property is a bit weird... Wouldn't it be better to make it private and initialize it from the constructor?